src/ShallowTraversal.js

```diff
@@ -27,6 +27,9 @@ export function childrenOfNode(node) {
 
 export function hasClassName(node, className) {
   let classes = propsOfNode(node).className || '';
+  if (typeof classes !== 'string') {
+    classes = classes.toString();
+	}
```

Comment:
Hey! Thanks for the PR! couple of things here:

1. I'm not familar with `bem-cn`, can you explain a bit about how it works? From my understanding, React expects `className` to be a string at render time. How does `bem-cn` properly pass a function to that?

2. Your identation is off.
3. We need unit tests or there is a good chance obscure code like this could be regressed.

You can hold off on executing 2 & 3 until we determine the necessity of this PR if you want. Thanks for starting the conversation around this!